### ISSUE TITLE
Add bundle of savings and investment vehicles 

### ISSUE THREAD

**Issue by kbottagaro** [Gary-Community-Ventures/benefits-calculator#1321]
@sydney-devine This is a backlog item. Just adding it here at Elise's request as a discussion point for a later date.

A couple of partners have asked us to include programs that require someone to save money and possibly have an investment risk (e.g., CollegeInvest First Step, ABLE Accounts). These programs fall outside of our current long-term and immediate needs resources. Do we want to create a category of programs that would appear if someone checked a box looking for savings or investment opportunities?

**Comment by sydney-devine** [comment]
Let's talk about this during a biz mtg?

**Comment by kbottagaro** [comment]
Yes - this came up in yesterday's biz meeting. This is truly a backburner
kind of topic - just capturing it in a ticket so we don't lose it.
